### DISCUSSION TITLE
Update appstream metadata

### DISCUSSION THREAD

**Discussion by 0xMRTT** [GradienceTeam/Gradience#101] [General]
Currently appstream metadata are empty. We need to add screenshots, link to the repo, etc...

https://freedesktop.org/software/appstream/docs/chap-Metadata.html

**Comment by 0xMRTT** [comment]
https://gitlab.gnome.org/GNOME/gnome-software/-/wikis/Software-metadata#user-content-how-to-set-the-carousel-tile-background-colour

**Comment by daudix** [comment]
Somthing like this?

[📎 Appdata.zip](https://github.com/AdwCustomizerTeam/AdwCustomizer/files/9311071/Appdata.zip)

**Reply by 0xMRTT** [reply]
Yes.

**Reply by daudix** [reply]
Soon I will make new screenshots and add it in Design repo, it will also used in appdata

**Reply by daudix** [reply]
And also I will make banner color override

**Comment by daudix** [comment]
Need suggestion, what screenshots will look better, normal or pretty purple?

**Reply by 0xMRTT** [reply]
Pretty purple and normal because pretty purple is the usage of the app. So maybe include pretty purple and default

**Reply by daudix** [reply]
Thank you.

**Comment by daudix** [comment]
Done, only thing left is a little changelog

**Reply by 0xMRTT** [reply]
I wrote it

**Reply by 0xMRTT** [reply]
@daudix-UFO could you review the Changelog ?

**Reply by daudix** [reply]
Of course, give me a minute!

**Reply by daudix** [reply]
Done! fixed a few moments ;)

**Reply by daudix** [reply]
Noticed that first screenshot is old, don't release for now

**Reply by daudix** [reply]
Done, screenshot fixed as well as few typo's
